H2MLauncher.UI/Model/GlobalKeyboardHook.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+
+namespace H2MLauncher.UI.Model
+{
+    public class GlobalKeyboardHook : IDisposable
+    {
+        private const int WH_KEYBOARD_LL = 13;
+        private const int WM_KEYDOWN = 0x0100;
+
+        private LowLevelKeyboardProc _proc;
+        private IntPtr _hookID = IntPtr.Zero;
+
+        public event Action<Key, ModifierKeys> KeyPressed;
+
+        public GlobalKeyboardHook()
+        {
+            _proc = HookCallback;
+            _hookID = SetHook(_proc);
+        }
+
+        private IntPtr SetHook(LowLevelKeyboardProc proc)
+        {
+            using (Process curProcess = Process.GetCurrentProcess())
```

Comment:
Make it like this:
```cs
using Process curProcess = Process.GetCurrentProcess();
using ProcessModule curModule = curProcess.MainModule!;
```
Because the process will and should always be available, so it won't be a null